lib/experimental/segyio/segyio.hpp

```diff
@@ -13,6 +13,7 @@
 #include <vector>
 
 #include <segyio/segy.h>
+#include <cstdint>
```

Comment:
This in theory should be near other common includes, not near segy.h one, but ok.

I even think other includes are sorted alphabetically, here and in the other file as well. But we can be evil and break sorting too here.

Reply:
I just added another fixup to reorder it. I want it to be in order. 😄